MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,643 @@
+---
+layout: post
+title: Working with DataForm in Maui DataForm control | Syncfusion
+description: Learn here all about Working with DataForm support in Syncfusion Maui DataForm (SfDataForm) control and more.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+
+# Working with DataForm in Maui DataForm (SfDataForm)
+
+## Auto-generating DataFormItems for the data field
+
+By default, the [DataFormItems]`Link` will be generated based on the property type. For example, the [DataFormTextItem]`Link` will be created for the `string` type property. 
+
+The [DataFormItem]`Link` generation depends on the type and attribute defined for the property.
+The following tables lists the several types of `DataFormItem` and its constraints for auto generation:
+
+<table>
+<tr>
+<th>Generated DataFormItem Type</th>
+<th>Editor</th>
+<th>Data Type / Attribute</th>
+</tr>
+<tr>
+<td>
+[DataFormTextItem]
+</td>
+<td>
+Text
+</td>
+<td>
+Default DataFormItem generated for the String type and the properties with [DataType(DataType.Text)], [DataType(DataType.MultilineText)] and [DataType(DataType.Password)] attributes. 
+
+</td>
+</tr>
+<tr>
+<td>
+[DataFormMultiLineTextItem]
+</td>
+<td>
+Text
+</td>
+<td>
+Generated for the String type and the properties with  [DataType(DataType.MultilineText)] attributes. 
+
+</td>
+</tr>
+<tr>
+<td>
+[DataFormPasswordItem]
+</td>
+<td>
+Text
+</td>
+<td>
+Generated for the String type and the properties with [DataType(DataType.Password)] attributes. 
+</td>
+</tr>
+
+<tr>
+<td>
+[DataFormDateItem]
+</td>
+<td>
+Date
+</td>
+<td>
+Generated for DateTime type and properties with [DataType(DataType.Date)] and [DataType(DataType.DateTime)] attributes. 
+
+</td>
+</tr>
+<tr>
+<td>
+[DataFormTimeItem]
+</td>
+<td>
+Time
+</td>
+<td>
+Generated for the property with [DataType(DataType.Time)] attribute. 
```

Comment:
Generated for the TimeSpan, TimeOnly, DateTime and DateTimeOffset property

Reply:
property types added